lookupapi/views.py

```diff
@@ -94,6 +93,21 @@ def list(self, request):
         }, context={'request': request}).data)
 
 
+def getPerson_or_404(scheme, identifier, fetch):
+    """
+    Gets a lookup entry for a Person with the given schema and identifier. If the person
+    scheme is "mock" then test user mug99 is returned
+    :param scheme: usually "crsid" but also "mock"
+    :param identifier: the id of the person for the given scheme
+    :param fetch: any extra fetch parameters to pass to ibis
+    :return:
+    """
+    if scheme == "mock":
```

Comment:
How do we configure production to not recognise mock people?

Reply:
Maybe the cleanest way is to have a ALLOWED_SCHEMES setting? Then production can have ALLOWED_SCHEMES = ['crsid', 'self'] and dev can have ALLOWED_SCHEMES = ['crsid', 'self', 'mock']?

Reply:
mug99 is also in the production lookup, so I don't see it as a huge problem.